views/layouts/MainLayout.jsx

```diff
@@ -34,8 +36,36 @@ const MainLayout = ({ classes, theme }) => {
     }
   }
 
+  const processDates = (options) => {
+    const momentSetting = {
+      sameDay: '[Today]',
+      nextDay: '[Tomorrow]',
+      nextWeek: 'dddd',
+      lastDay: '[Yesterday]',
+      lastWeek: '[Last] dddd',
+      sameElse: 'MM/DD/YYYY',
+    }
+    const nowT = moment().local()
```

Comment:
How tricky is it to do this with ES6? Moment.js doesn't actually recommend people use their library anymore, now that JS is more mature: https://momentjs.com/docs/#/-project-status/

Fun fact, I learned this when I told Alex my favorite library was moment.js in my Gnar interview.

Reply:
I'm not sure, but I am thinking of changing this entirely as I don't think that we really need it. I remember the moment when moment said goodbye and it recommended a more modern solutions that were available at that moment and our current moment.